rfc/jira-translator-cdevents.md

```diff
@@ -19,9 +19,588 @@ Issue related Jira events can be mapped to CDEvent's [Ticket Events](https://cde
+| dev.cdevents.ticket.created| jira:issue_created |
+<details>
+<summary>Event JSON</summary>
+
+```
```

Comment:
I tried this, not sure it is displaying in the right format. 
When I view the file,
https://github.com/Nordix/cdevents_community/blob/15363c42d2ea3a0fbbaa5a3bddf08df76e83f35c/rfc/jira-translator-cdevents.md

Reply:
It looks right? What do you mean by "right" format?

Reply:
![image](https://github.com/Nordix/cdevents_community/assets/97948659/495e5ddd-a9d3-4272-91c0-36446b94c579)

Reply:
OH! hahaha, it escaped it. Let me fix that